rerun_py/rerun_sdk/rerun/_rerun2/archetypes/pinhole_ext.py

```diff
@@ -0,0 +1,110 @@
+from __future__ import annotations
+
+from typing import Any, cast
+
+import numpy.typing as npt
+
+from rerun.log.error_utils import _send_warning
+
+from ..datatypes.mat3x3 import Mat3x3Like
+from ..datatypes.vec2d import Vec2D, Vec2DLike
+
+
+class PinholeExt:
+    """
+    Log a perspective camera model.
+
+    Parameters
+    ----------
+    image_from_cam:
+        Column-major projection matrix.
+
+        Child from parent.
+        Image coordinates from camera view coordinates.
+
+        Example:
+        ```text
+        [[1496.1, 0.0,    0.0], // col 0
+        [0.0,    1496.1, 0.0], // col 1
+        [980.5,  744.5,  1.0]] // col 2
+        ```
```

Comment:
Please no. Python numpy ArrayLike constructor conventions are row-major.

The row-to-col major swap happens here, which should happen transparently down inside of `__attrs_init__`
https://github.com/rerun-io/rerun/blob/ddb6eac02405c30363d88e251f5f8edd17cb8c6b/rerun_py/rerun_sdk/rerun/_rerun2/datatypes/mat3x3_ext.py#L17-L21

Reply:
This should continue to match the conventions from:
https://github.com/rerun-io/rerun/blob/ddb6eac02405c30363d88e251f5f8edd17cb8c6b/rerun_py/rerun_sdk/rerun/log/camera.py#L77-L81

Reply:
Comment is wrong. Python ingestion *is* still row-major but this comment was incorrecetly copy pasted from Rust